drivers/hmis/app/models/hmis/ce/referral_participant.rb

```diff
@@ -1,10 +1,21 @@
 # frozen_string_literal: true
 
-# A role/user who is participating in the referral process
+# A role/user who is participating in the referral process.
```

Comment:
@gigxz following up on our frontend conversation, I've tried to add some documentation here about the difference between ReferralParticipants and StepAssignments.

This raised a couple more questions for me about the desired functionality:
- Is it correct that, when you start a task, you are assigned but existing participants stay assigned too? Or should they be unassigned? 
- Should there be some more confirmation around starting tasks?